Sources/tart/Commands/Run.swift

```diff
@@ -89,17 +89,20 @@ struct Run: AsyncParsableCommand {
   var rosettaTag: String?
 
   @Option(help: ArgumentHelp("""
-  Additional directory shares with an optional read-only specifier\n(e.g. --dir=\"~/src/build\" or --dir=\"~/src/sources:ro\")
+  Additional directory shares with a comma separated list of options\n(e.g. --dir=\"~/src/build\" or --dir=\"~/src/sources:ro\")
```

Comment:
You can pass multiple `--dir`:

```
tart run sonoma-base --dir="~/src/build" or --dir="~/src/sources:ro"
```

Not only comma separated.

Reply:
The "comma separated" part refers to the list of options, not the list of directories:

```
tart run sonoma-base --dir="~/src/build:ro" or --dir="sources:~/src/sources:ro,noauto"
```

Reply:
How about "Additional directory shares, each with a comma separated list of options" ?